src/style.css

```diff
@@ -151,4 +152,431 @@ nav.active a img {
   nav ul li a:focus img {
     transform: scale(1.1);
   }
+
+  /* desktop Menu update */
+  .body_container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    margin-left: 0;
+  }
```

Comment:
Once you've made the requested changes, please revisit this particular section of the code to ensure that the modifications do not adversely affect the final outcome. It's important to maintain the overall functionality and integrity of the project.